lib/merkle_tree.ex

```diff
@@ -22,10 +22,13 @@ defmodule MerkleTree do
 
   defstruct [:blocks, :root, :hash_function]
 
-  @number_of_children 2 # Number of children per node. Configurable.
+  @number_of_children 2 # Number of children per node
+  @default_height 16
+  @zeroes <<0>> |> List.duplicate(32) |> Enum.join
 
   @type blocks :: [String.t, ...]
   @type hash_function :: (String.t -> String.t)
+  @type height :: non_neg_integer
```

Comment:
Height is number, not ordinal; so always positive and minimal value is 1

Reply:
I.e. MerkleTree.new(..,.., 0) should crash

Reply:
As we speaked, let's leave it this way.